frontend/src/App.css

```diff
@@ -3,3 +3,70 @@
   padding: 0;
   box-sizing: border-box;
 }
+body {
+  background-color: #066acd;
+}
+.App {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 5%;
+}
+.card {
+  padding: 2% 3%;
+  width: 30%;
+  background-color: white;
+  border-radius: 5%;
+}
+input:hover {
+  border-color: #24a0ed;
+}
+input {
+  width: 100%;
+  padding: 3%;
+  margin: 1%;
+}
+ul {
```

Comment:
You are still using element selectors like `input` and `ul`. This means that all the input fields and all unordered lists will have the same style. We do not generally want to use the same style for all instances of an element.

Replace these with a class and add it to the Todo input and list elements.